packages/plugins/src/foundry/index.ts

```diff
@@ -369,20 +430,138 @@ const command = args[0]
       const { parsedConfig, minimalParsedConfig, configArtifacts } =
         await readUnvalidatedParsedConfig(configPath, cre, getConfigArtifacts)
 
-      // TODO: We should write parsedConfig and configArtifacts to a 'chugsplash-cache' file or
-      // something. We'll need them in the next ffi call, `postParsingValidation`
+      const configCache = {
+        parsedConfig,
+        configArtifacts,
+      }
+
+      if (!fs.existsSync('./cache')) {
+        fs.mkdirSync('./cache')
+      }
+      fs.writeFileSync(
+        './cache/chugsplash-config-cache.json',
+        JSON.stringify(configCache, null, 2),
+        'utf-8'
+      )
+
+      const ChugSplashFoundryABI =
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        require('../../out/artifacts/ChugSplash.sol/ChugSplash.json').abi
```

Comment:
How about we use `forge inspect` here? I think it'd be a little less brittle in case we ever change the artifact folder.
```
const abiRaw = await execAsync('forge inspect ChugSplash abi')
const abi: Array<Fragment> = JSON.parse(abiRaw.stdout)
```
Same thing for the other two places we get the ChugSplashFoundryABI

Reply:
This seems extremely slow in some situations, I think because it's potentially triggering a rebuild. I.e if I run `forge build` and then run this command it takes 30 seconds before it outputs anything despite the fact that the artifact should be readily available. Doesn't seem worth using it to me.

Reply:
Ah i see. What about running `execAsync(forge config --json)`, like in `getPaths`, to get the "out" folder?

Reply:
In fact I think we can use the `artifactFolder`, which is already available